streams/readable-byte-streams/general.any.js

```diff
@@ -337,8 +337,8 @@ promise_test(() => {
   return Promise.resolve().then(() => {
     assert_equals(pullCount, 1, 'pull() must have been invoked once');
     const byobRequest = byobRequests[0];
-    assert_true(byobRequest.defined, 'first byobRequest must not be undefined');
-    assert_true(byobRequest.viewDefined, 'first byobRequest.view must not be undefined');
+    assert_true(byobRequest.nonNull, 'first byobRequest must not be null');
+    assert_true(byobRequest.viewDefined, 'first byobRequest.view must not be null');
```

Comment:
`byobRequest.viewDefined` is set to `view !== undefined`, so the message should be "first byobRequest.view must not be undefined" (as it was before).

Reply:
Seems like this also needs to become nullable.